MAT.SQLRace.HelloData/Program.cs

```diff
@@ -849,6 +848,61 @@ private static void AddAndProcessMarkers()
             }
         }
 
+        /// <summary>
+        /// Load data an SSN file, add some Markers to it if they don't already exist, extract the data in the ranges defined by the Markers.
+        /// </summary>
+        private static void AddAndProcessTransientMarkers(string connectionString)
+        {
+            try
+            {
+                var sessionManager = SessionManager.CreateSessionManager();
+
+                // "E720632B-B135-4409-8E1F-ECFC76BD716A"
+                const string sessionKey = "e720632b-b135-4409-8e1f-ecfc76bd716a";
+
+                var clientSession = sessionManager.Load(new SessionKey(sessionKey), connectionString);
+
+                if (clientSession == null)
+                {
+                    Console.WriteLine("Session not found");
+                    return;
+                }
+
+                var session = clientSession.Session;
+
+                const string markerType = "MARKER";
+
+                var newMarkers = new List<Marker>();
+
+                // Add some demo markers to the sessions to define some arbitrary time ranges
+                for (var i = 1; i <= 3; i++)
+                {
+                    // Start in the middle of the session and mark 20ms every 100s
+                    var markerStartTime = session.StartTime + ((session.EndTime - session.StartTime) / 2) + (i * 100 * NanosecondExtensions.NanosecondsPerSecond);
+                    // 20ms after the start time
+                    var markerEndTime = markerStartTime + (20 * NanosecondExtensions.NanosecondsPerMillisecond);
+
+                    var marker = Marker.CreateTransientMarker(markerStartTime, markerEndTime, "Tansient Marker" + i, markerType, "Demonstration Transient marker " + i);
```

Comment:
Adding a comment somethwhere to explain that you can also add transient point markers would be nice.

Reply:
I don't think this is in scope of the original ticket - all descriptions are on the parent ticke in ADO